pkg/object/meshcontroller/worker/worker.go

```diff
@@ -70,28 +70,24 @@ type (
 	}
 )
 
-func decodeLabels(labels string) map[string]string {
-	mLabels := make(map[string]string)
-	if len(labels) == 0 {
-		return mLabels
-	}
-	strLabel, err := url.QueryUnescape(labels)
-	if err != nil {
-		logger.Errorf("query unescape: %s failed: %v ", labels, err)
-		return mLabels
+func decodeLabels(labelStr string) map[string]string {
+	labelMap := make(map[string]string)
+	if len(labelStr) == 0 {
+		return labelMap
 	}
 
-	arrLabel := strings.Split(strLabel, "&")
+	labelSlice := strings.Split(labelStr, ",")
```

Comment:
This will not break existing configs, right?

Reply:
The older one is not idiomatic and weird in the config, this aims to fix it. I had thought about replacing `&` with `,` in the beginning of the function. But I didn't do it to bring more historic stuff.